src/screens/Details/styles/index.ts

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
```

Comment:
there is really no need in separate styles folder. It can simply be `styles.ts` file